src/Parse/ParseClient.php

```diff
@@ -255,7 +260,7 @@ public static function _request($method, $relativeUrl, $sessionToken = null,
         self::assertParseInitialized();
         $headers = self::_getRequestHeaders($sessionToken, $useMasterKey);
 
-        $url = self::HOST_NAME . '/' . ltrim($relativeUrl, '/');
+        $url = self::HOST_NAME . '/' . self::API_VERSION . '/' . ltrim($relativeUrl, '/');
```

Comment:
I see no point on calling to another static method of the same class when you can perfectly compose the URL here. This is done only in one place, If it were used in several more places then I'd see a real benefit